resource-definitions/sconfig-example3.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: "secretless.io/v1"
+kind: Configuration
+metadata:
+  name: secretless-example-config2
+spec:
+  listeners:
+    - name: http_config_1_listener
```

Comment:
do you want the `config_1` in the listener / handler names to be `config_2` here (and below)?